### ISSUE TITLE
[SCREENREADER]: Form submit error alert is missing an "alert" role

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#11080]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The form system shows an alert when there is an issue connecting to the API

![Screen Shot 2020-07-09 at 11 17 58 AM](https://user-images.githubusercontent.com/136959/87066464-3c7ad100-c1d8-11ea-9180-39d280188f30.png)

The alert is not announced by the screenreader, and is missing a `role="alert"`

## Acceptance Criteria

As a screenreader user, I want to hear the contents of an alert when it is shown.

## Environment

All; tested on MacOS Safari & VoiceOver

## Steps to Recreate

Found while working in a local dev environment with the Higher-Level Review form. It is not reproducible on staging due to back-end API issues that need to be resolved.

This could be reproduced with any form if the internet was disabled prior to submission

<details><summary>How to go Offline</summary>

1. Open the dev tools
2. Go to the "Network" tab
3. Set the Throttling dropdown to "Offline"
  ![offline](https://user-images.githubusercontent.com/136959/87067219-5ff24b80-c1d9-11ea-9866-801e0d1f565b.gif)
</details>

## Possible Fixes (optional)

Add `role="alert"` to the submit buttons component
